src/cli/consts.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+export const APP_NAME = 'llama-pack-cli';
```

Comment:
What are our current thoughts about including the llama-pack version as well?

Reply:
I think we want to add it at some point. Not sure what's the best approach right now though (Do we want the llama-pack version, the template version or most likely both?). So punting this bit until we have more clarity